Installing from source fails due to unsatisfied statik dependency

Trying to follow the readme instructions for installing from source, but hit:

```
$ make install
cd pkg/themis-contract/ && statik -src=../../assets/
/bin/sh: statik: command not found
make: *** [Makefile:11: build] Error 127
``

Also failing on my system (manjaro) because of the chosen install directory:

```
$ make install
cd pkg/themis-contract/ && statik -f -src=../../assets/
go build -mod=readonly -ldflags="-X main.version=v0.1.5-153-g9eb5cba-20200908.190928" -o build/themis-contract cmd/themis-contract/*
go get github.com/rakyll/statik
cp build/themis-contract /usr/local/bin/
cp: cannot create regular file '/usr/local/bin/themis-contract': Permission denied
make: *** [Makefile:19: install] Error 1
```

I'm wondering why we can't install it as a go package?

--- a/Makefile
+++ b/Makefile
@@ -15,6 +15,7 @@ test:
 	go test $(PACKAGES)
 
 install: build
+	go get github.com/rakyll/statik
 	cp $(OUTPUT) /usr/local/bin/
 
 .PHONY: all build test install

--- a/Makefile
+++ b/Makefile
@@ -5,6 +5,8 @@ TIMESTAMP=$(shell date -u +%Y%m%d.%H%M%S)
 BUILD_FLAGS=-mod=readonly -ldflags="-X main.version=$(VERSION)-$(TIMESTAMP)"
 .DEFAULT_GOAL := build
 
+THEMIS_INSTALL_DIR ?= /usr/local/bin/
+
 all: build test
 
 build:
@@ -16,6 +18,6 @@ test:
 
 install: build
 	go get github.com/rakyll/statik
-	cp $(OUTPUT) /usr/local/bin/
+	cp $(OUTPUT) $(THEMIS_INSTALL_DIR)
 
 .PHONY: all build test install

--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ BUILD_FLAGS=-mod=readonly -ldflags="-X main.version=$(VERSION)-$(TIMESTAMP)"
 all: build test
 
 build:
-	cd pkg/themis-contract/ && statik -src=../../assets/
+	cd pkg/themis-contract/ && statik -f -src=../../assets/
 	go build $(BUILD_FLAGS) -o $(OUTPUT) cmd/themis-contract/*
 
 test:

--- a/Makefile
+++ b/Makefile
@@ -9,6 +9,9 @@ THEMIS_INSTALL_DIR ?= /usr/local/bin/
 
 all: build test
 
+deps:
+	go get github.com/rakyll/statik
+
 build:
 	cd pkg/themis-contract/ && statik -f -src=../../assets/
 	go build $(BUILD_FLAGS) -o $(OUTPUT) cmd/themis-contract/*
@@ -17,7 +20,6 @@ test:
 	go test $(PACKAGES)
 
 install: build
-	go get github.com/rakyll/statik
 	cp $(OUTPUT) $(THEMIS_INSTALL_DIR)
 
-.PHONY: all build test install
+.PHONY: all build test install deps

--- a/README.md
+++ b/README.md
@@ -41,6 +41,8 @@ To rather install from source:
 ```bash
 git clone https://github.com/informalsystems/themis-contract.git
 cd themis-contract
+# once-off
+make deps
 # Setting THEMIS_INSTALL_DIR to our desired location
 # (default is to /usr/local/bin/)
 THEMIS_INSTALL_DIR=~/.local/bin make install

--- a/README.md
+++ b/README.md
@@ -36,11 +36,14 @@ latest [release] binary for your platform (right now we only build for Linux
 and MacOS) and put it somewhere in your path (e.g.
 `/usr/local/bin/themis-contract`).
 
-To rather install from source, clone this repository locally and simply run:
+To rather install from source:
 
 ```bash
-# Will build and install to /usr/local/bin/themis-contract
-make install
+git clone https://github.com/informalsystems/themis-contract.git
+cd themis-contract
+# Setting THEMIS_INSTALL_DIR to our desired location
+# (default is to /usr/local/bin/)
+THEMIS_INSTALL_DIR=~/.local/bin make install
 ```
 
 ## Usage

